app/views/movies/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<h2>Welcome <%= current_user.first_name %> <%= current_user.last_name %></h2>
+
+<h1><%= @movie.title %></h1>
+
+<section id='details'>
+  <p>
+    <b>Vote Average:</b> <%= @movie.vote_average %>
+    <b>Runtime:</b> <%= @movie.runtime_convertion %>
+    <b>Genre(s):</b> <%= @movie.get_genres %>
+  </p>
+</section>
+
+<section id='summary'>
+  <p>
+    <b>Summary</b><br>
+    <%= @movie.overview %>
+  </p>
+</section>
+
+
+<b>Cast</b><br>
+<% if @movie.cast != []%>
```

Comment:
Just curious, is this data sometimes missing so this is empty?

Reply:
We discussed, and sometimes the information is missing.

Reply:
Sometimes the cast, genres, and reviews can be empty